Adding support for the <a> HTML tag

cesarsouza opened the pull request:
Hi there,

I am sending a pull request to add support for the HTML Anchor Element (&lt;a&gt;). It is supported as an extended XML documentation tag by Visual Studio/IntelliSense, Sandcastle, and many others. 

For example, consider this [source code for a NormalDistribution implemented in the Accord.NET Framework](https://github.com/accord-net/framework/blob/development/Sources/Accord.Statistics/Distributions/Univariate/Continuous/NormalDistribution.cs#L70). At [line 70](https://github.com/accord-net/framework/blob/development/Sources/Accord.Statistics/Distributions/Univariate/Continuous/NormalDistribution.cs#L70) the tag <a href="..." /> is used to link against a Wikipedia article. You can see it is totally supported by Sandcastle by checking the [Remarks section in the generated page](http://accord-framework.net/docs/html/T_Accord_Statistics_Distributions_Univariate_NormalDistribution.htm) at 

http://accord-framework.net/docs/html/T_Accord_Statistics_Distributions_Univariate_NormalDistribution.htm

Hope it helps!

Regards,
Cesar

cesarsouza committed:
Adding support for the <a> HTML tag. It is supported as an extended documentation tag by Visual Studio, Sandcastle, and many others.

cesarsouza committed:
Adding support for reading Source and Region attributes from Code tags.

cesarsouza commented on src/NuDoq/Properties/AssemblyInfo.cs:
```diff
@@ -41,4 +41,6 @@ partial class ThisAssembly
     public const string Owners = "kzu";
     public const string Authors = "Daniel Cazzulino, kzu";
+
+    public const string Version = "https://github.com/cesarsouza/NuDoq";
```
Sorry, after almost 3 years I do not remember what that was needed :-) I think it was because I needed to distribute my version within a particular software, and I needed to refer to the exact version  I was distributing - which happened to be my fork instead of yours.

Please feel free to modify it if you would still be willing to incorporate those change into your fork.

Regards,
Cesar

kzu replied:
Makes sense! Sorry to bother you after so long :(

cesarsouza committed:
Removing version tag

cesarsouza commented:
Hi,

I've removed that version tag from the commit using GitHub's online editor.

By the way, I had added another feature to my own branch at https://github.com/cesarsouza/NuDoq if you have interest in continuing this project. I had added support for reading source and region attributes from code tags, as shown in this commit: https://github.com/cesarsouza/NuDoq/commit/4d7a12450cd4877fd29c95e89f8730c3626fd9e4

This is also another not very well documented feature, but basically you can insert code snippets into documentation by pointing to a source file and a #region/#endregion block inside this file. 

That commit adds support for parsing those.

You have a very useful project here, thanks a lot for sharing it in the first place!

Regards,
Cesar

kzu commented:
I love what you're doing. Added you as a collaborator directly so you can push directly to this repo 👍 

Thanks!

cesarsouza commented:
Hi @kzu,

Thanks, I will try to find some time to integrate at least my changes. However, I am not sure I will be able to help maintain the project in the long term (I have already a couple ones to take care of). But I can certainly try to integrate my own stuff.

They were mostly used to create [Statistics Workbench](https://www.codeproject.com/Articles/835786/Statistics-Workbench) a couple years ago.

Regards,
Cesar

kzu commented:
He, that sounds like it was a really fun project to work on! 👍 

I'm also super tied on time, so no hurries if you can, and no worries if you can't ;)

Thanks!

kzu commented:
Closing for now until Cesar can hopefully contribute back :)
